resources/geometry.rb

```diff
@@ -61,6 +124,28 @@ def self.get_height_of_spaces(spaces)
     end
     return maxzs.max - minzs.min
   end
+  
+  def self.get_height_of_thermal_zone(building:,
+                                      thermal_zone:)
+    heights = []
+    building.elements.each("BuildingDetails/Enclosure/FoundationWalls/FoundationWall") do |foundation_wall|
+      foundation_wall_values = HPXML.get_foundation_wall_values(foundation_wall: foundation_wall)
+      next if foundation_wall_values[:interior_adjacent_to] != thermal_zone
+
+      height = foundation_wall_values[:height]
+      if building.elements["BuildingDetails/Enclosure/RimJoists/RimJoist"]
+        height += 1.0
+      end
+      heights << height
+    end
+    building.elements.each("BuildingDetails/Enclosure/Walls/Wall") do |wall|
+      wall_values = HPXML.get_wall_values(wall: wall)
+      next if wall_values[:interior_adjacent_to] != thermal_zone
+
+      heights << 8.0
```

Comment:
Same comment as above.

Reply:
Seems like this needs to incorporate the number of above-grade stories for 2+ story homes?

Reply:
`Constants.WallHeight`?

Reply:
Is "living space" the only thermal zone that can have more than one story?

Reply:
Yes.